zerver/lib/test_helpers.py

```diff
@@ -328,7 +327,6 @@ def __init__(
         self.user = user_profile or AnonymousUser()
         self._body = b""
         self.content_type = ""
-        BaseNotes[str, str].get_notes
```

Comment:
For this commit, can you add to the commit message that it was introduced in 53888e5a26ce0ebfc3cb56877cee363a3d9790d9 in error?  Generally when we see dead code, it's worth checking and confirming how it got there, in case the problem was something else. (Like we were supposed to do something here).